### PR TITLE
[CIS-511] Update `addReaction` API with `enforce_unique` parameter

### PR DESCRIPTION
**Pull request by mantergo**
In this PR:

- Update `addReaction` API with `enforce_unique` parameter

### CODE REVIEW

File: Sources/StreamChat/Workers/MessageUpdater.swift [hunk 1]
```diff
@@ -241,19 +241,22 @@ class MessageUpdater<ExtraData: ExtraDataTypes>: Worker {
...
     ///   - extraData: The extra data attached to the reaction.
     ///   - messageId: The message identifier the reaction will be added to.
+    ///   - enforceUnique: If set to `true`, new reaction will replace all reactions the user has (if any) on this message.
     ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
     func addReaction(
         _ type: MessageReactionType,
         score: Int,
         extraData: ExtraData.MessageReaction,
         messageId: MessageId,
+        enforceUnique: Bool = false,
```
Comment (VojtaStavik) [inline Sources/StreamChat/Workers/MessageUpdater.swift:251]: This is an updater, so the default parameter here is not needed. Did I miss something or is the public API for this feature missing? :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # Upcoming
 
 ### ✅ Added
+- Add support for `enforce_unique` parameter on sending reactions
```
Comment (VojtaStavik) [inline CHANGELOG.md:10]: This is an LLC change, so you need to use the changelog located in `Sources/StreamChat`

**Review by VojtaStavik — CHANGES_REQUESTED** [review]
I think you forgot the public API part of this feature :)

File: CHANGELOG.md (hunk 2 above)
Comment (VojtaStavik) [reply CHANGELOG.md:10]: .... and we have just one CHANGELOG again, so ignore this :)

File: Sources/StreamChat/Workers/MessageUpdater.swift (hunk 1 above)
Comment (mantergo) [reply Sources/StreamChat/Workers/MessageUpdater.swift]: Idk how this happened and don't have an excuse 😳
Fixed!

**Review by VojtaStavik — APPROVED** [review]

**Pull request merged**